src/VirtualMonoRepo/VirtualMonoRepo.Tasks.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <RunAnalyzers>false</RunAnalyzers>
+    <Nullable>disable</Nullable>
+    <LangVersion>preview</LangVersion>
+    <SignAssembly>false</SignAssembly>
+    <IsPackable>false</IsPackable>
+    <IsShipping>false</IsShipping>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="*.cs" />
+
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkVersion)" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildUtilitiesCoreVersion)" />
+    <!-- TODO (https://github.com/dotnet/arcade/issues/10492): Set the version from Versions.props -->
```

Comment:
Why not fix this before checkin?

Reply:
This is something I wanted to discuss - I am not sure whether we want a subscription from `arcade-services` into `installer`. Would it be a problem in any way?

Reply:
This is a better question for @mmitche

Reply:
I have pulled the version into `Versions.props` file regardless and I will talk to Matt whether we want to flow the DarcLib version automatically or not

Reply:
We can have that subscription. The dependency should go into the Toolset section of the Version.Details.xml

Reply:
Great! Done

Reply:
@mmitche I found out that `dotnet/arcade-services` publishes main builds to the `.NET Eng Services - Int` channel. Is it okay to create a subscription for this one? Not sure what is its purpose